src/xdp/precomp.h

```diff
@@ -12,7 +12,27 @@
 #include <ntifs.h>
 #include <ntintsafe.h>
 #include <ndis.h>
+//
+// The stdint.h header is included by eBPF headers, and stdint.h throws
+// warnings. Include it directly and suppress the warnings.
+//
+// TODO: Attempt to eliminate the warnings.
```

Comment:
Would this require changing `stdint.h`? If so, I doubt you're going to be able to get that changed, so I'd just remove this TODO.

Reply:
Yeah, this doesn't seem tractable in the short term. I've removed the TODO.